examples/datasource-ciphered-password/README_pt.adoc

```diff
@@ -52,7 +52,7 @@ public static class ReverseEncryption implements PasswordCipher {
 Para ser funcional, ele precisa do arquivo
 `META-INF/org.apache.openejb.resource.jdbc.PasswordCipher/reverse`.
 
-O nome do arquivo (reverso) define o nome da criptografia a ser usado para o
```

Comment:
Hi Daniel, good translation, with just one point of attention, this sentence "nome da criptografia a ser usado" to "nome da criptografia a ser usada"

Reply:
Done